src/main/resources/bidder-config/generic.yaml

```diff
@@ -18,3 +18,14 @@ adapters:
         - audio
       supported-vendors:
       vendor-id: 0
+      cwire:
+        enabled: false
+        endpoint: https://ortb.cwi.re/v1/bid
+        modifying-vast-xml-allowed: false
+        endpoint-compression: gzip
+        meta-info:
+          maintainer-email: devs@cwire.com
+          site-media-types:
+            - banner
+          supported-vendors:
+          vendor-id: 1081
```

Comment:
Incorrect way to add an alias. Refer to [prebid java docs](https://docs.prebid.org/prebid-server/developers/add-new-bidder-java.html#create-bidder-alias) for more instructions